server/utils/conditionsProviderExpand.test.ts

```diff
@@ -0,0 +1,151 @@
+import { expandAdditionalConditions } from './conditionsProvider'
+import { AdditionalCondition } from '../@types/licenceApiClientTypes'
+
+describe('Conditions Provider - expansions', () => {
+  describe('expandAdditionalConditions', () => {
+    it('Will accept and return an empty list', () => {
+      const conditions: AdditionalCondition[] = []
+      const listOfConditions = expandAdditionalConditions(conditions)
+      expect(listOfConditions).toHaveLength(0)
+    })
+
+    it('Will return text verbatim when no placeholders exist', () => {
+      const conditions: AdditionalCondition[] = [
+        {
+          id: 1,
+          code: 'ed607a91-fe3a-4816-8eb9-b447c945935c',
+          category: 'Possession, ownership, control or inspection of specified items or documents',
+          sequence: 1,
+          text: 'Not to own or use a camera without the prior approval of your supervising officer.',
+          data: [],
+        },
+      ]
+      const listOfConditions = expandAdditionalConditions(conditions)
+      expect(listOfConditions).toHaveLength(1)
+      expect(listOfConditions[0]).toEqual(conditions[0].text)
```

Comment:
Always inclined to use string literals over variables in assertions

Reply:
In this case, its taking the .text verbatim - so should be identical. Agree where there are changes being made.